### PR TITLE
add SoftclDiceLoss and SoftDiceclDiceLoss loss function in documentation

### LINKED ISSUE #7193: cldice loss documentation
follow up of @jocpae in https://github.com/Project-MONAI/MONAI/issues/5938#issuecomment-1792441210, the relevant docstring should be reviewed and added to https://github.com/Project-MONAI/MONAI/blob/dev/docs/source/losses.rst?plain=1

please follow the instructions for changing the docs: https://github.com/Project-MONAI/MONAI/blob/dev/CONTRIBUTING.md#building-the-documentation

### PR DESCRIPTION
**Pull request by ytl0623**
Fixes #7193

### Description
add SoftclDiceLoss and SoftDiceclDiceLoss loss function in documentation(https://github.com/Project-MONAI/MONAI/blob/dev/docs/source/losses.rst?plain=1)

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Integration tests passed locally by running `./runtests.sh -f -u --net --coverage`.
- [ ] Quick tests passed locally by running `./runtests.sh --quick --unittests  --disttests`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated, tested `make html` command in the `docs/` folder.

### CODE REVIEW

File: docs/source/losses.rst [hunk 1]
```diff
@@ -137,3 +137,13 @@ Loss Wrappers
 ~~~~~~~~~~~~~~~~~~~~~
 .. autoclass:: DeepSupervisionLoss
     :members:
+
+`SoftclDiceLoss`
+~~~~~~~~~~~~~~~~~~~~~
```
Comment (KumoLiu) [inline docs/source/losses.rst:142]: Thanks for the PR.
Here should be aligned with the class name and also line 147.
Also, these two losses should be moved to the `Segmentation Losses` part.
https://github.com/Project-MONAI/MONAI/blob/eba3b52acc29054ae8ea29d64674ead132602f5b/docs/source/losses.rst#segmentation-losses

Comment (ytl0623) [reply docs/source/losses.rst]: Hi @KumoLiu, Thank you for your help. I have completed the correction and please help me review it again.
Thanks!

**Review by KumoLiu — APPROVED** [review]
LGTM now, thanks!

**Pull request merged**